tests/ips-state-actions-test-a-01/test.yaml

```diff
@@ -0,0 +1,25 @@
+pcap: ../tls/tls-certs-alert/input.pcap
+
+args:
+  - --set stream.midstream=false
+  - -k none
+
```

Comment:
isn't it important to run these as `IPS`?

Reply:
they are, because ips is in the test name

Reply:
Wait, if the test says ips-state SV will run them as IPS? 🤯